news/_posts/releases/2020-12-03-prestashop-1-7-7-0-available.md

```diff
@@ -0,0 +1,213 @@
+
+**Back-office**
+- [Adding incorrect language causes exception in Edit Currency page](https://github.com/PrestaShop/PrestaShop/issues/21891)
+- [Bad URL for module catalog page after disabling MBO](https://github.com/PrestaShop/PrestaShop/issues/21588)
+
+
+## Acknowledgments
+
+PrestaShop is above all a community project: from the 119 committers who contributed to this release, 57 are new contributors. The vast majority of PrestaShop contributors are not directly affiliated with the PrestaShop company.
+
+@formarion
+All contributors:
+123monsite-regis, 202 ecommerce, Abdullah, Adib Aroui, Aitbella Mohamed, Alexis Haumaitre, Amazzing, andromaque, Antoine Damiron, Antoine Thomas, Ashish Sharawat, Aude, Aurélien Pelletier, Benjamin, Benjamin Dussouillez, Boubker Bribri, Christian Kubitza, Christophe Zarebski, cirykpopeye, Clotaire Renaud, Codencode, ComonSoft, Damian Dominella, Daniel Hlavacek, Daniel Ziegenberg, Darius Aleksiunas, David Gonzalez, Dheeraj Sharma, Dinesh Badrukhiya, Dmitry, Florentin Garnier, Florian Bergeron, Florian Le Gars, Florian Lemaitre, Franck Lefèvre, François Peyret, Gavin Kalikapersaud, hacchus, Harlock, Horia Rudan, Ibrahima Sow, idnovate, JBWModules, Jean-François Viguier, Jevgenij Visockij, Jocelyn Fournier, Jonas Erixon, Jonathan François, Jonathan Lelievre, Jonathan Vollebregt, Julian Eberius, Julien Gissinger, Julius Žukauskas, Justinas Urbanavicius, Karel Faille, Khouloud Belguith, Klemart3D, Krystian Podemski, ks129, Laurynas Sedys, Louise Bonnard, Luc Vandesype, Manfredi Petruso, Marek Hanuš, Marion François, Marvin Sauraye, Mateusz Furga, Mathias Reker, Mathieu Ferment, Matthias Raigne, Matthieu Rolland, Maxim Krizhanovsky, Mehdi Badrani, Michael Voříšek, Mickaël Andrieu, mushroot, Nesrine Abdmouleh, okom3pom, Pablo Borowicz, Paulo Baptista, Peeyush Agrawal, Pierre Rambaud, pojebunny, Presta Module, Prestashark.eu, Prestaworks, PrestaworksNiklas, PululuK, Puma, Raimondas Sapola, Raúl Jiménez, Rinku Kazeno, Rodrigo Laurindo, Rokas Zygmantas, Rolige eCommerce Solutions, Roman Ondráček, seleda, Sergio Quiñonez, Simon Garny, Simone, Stephane Decisy, Sylvestre Nicky, Sébastien Bareyre, Tadas Davidsonas, Tanguy Salmon, Thomas Baccelli, Thomas L'huillier, Thomas Leviandier, Thomas Nabord, Tomas Ilginis, Tuni-Soft, Valentin Szczupak, venditdevs, Vincent Hadjedj, Vladimir, Web Premiere, webmak, Yannick Armand, Šarūnas Jonušas
+
+New contributors:
+Abdullah	39679397+OneDotIT@users.noreply.github.com	users.noreply.github.com
+Adib Aroui	aroui.adib@gmail.com	gmail.com
+Aitbella Mohamed	modfayz@gmail.com	gmail.com
+Alexis Haumaitre	alexis.haumaitre@prestashop.com	prestashop.com
+Antoine Damiron	damironantoine@gmail.com	gmail.com
+Ashish Sharawat	ashishsharawat8@gmail.com	gmail.com
+Aude	aude@bm-services.com	bm-services.com
+Benjamin	blopes@webxy.com	webxy.com
+Benjamin Dussouillez	bdussouillez@202-ecommerce.com	202-ecommerce.com
+Christophe Zarebski	kzarebski@digitalgarden.fr	digitalgarden.fr
```

Comment:
Do we want to display new contributors' email addresses?

Reply:
the answer is no

Reply:
No :sweat_smile: